src/style.css

```
@@ -1,3 +1,87 @@
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+  --active-color: #00acdc;
 }
+
+.nav__link {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  text-decoration: none;
+  color: #000;
+  font-size: 12px;
+  height: 60px;
+}
+
+.nav__link:hover {
+  color: var(--active-color);
+}
+
+.head {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+}
+
+.nav__logo {
+  margin-left: 50px;
+  height: 40px;
+}
+
+.nav__link.is-active::after {
+  content: "";
+  display: block;
+  position: relative;
+  top: 21px;
+  height: 4px;
+  background-color: var(--active-color);
+  border-radius: 8px;
+}
+
+span.is-active {
```

1. try to avoid using tags. better create one more class

2. don't increase specificity
```suggestion
.is-active {
```